File: RhythmType.java
public enum RhythmType {
    DOT(2),
    WHOLE_NOTE(4.0),
    HALF_NOTE(2),
    EIGHTH_NOTE_FIVELET(2/5),
    QUARTER_NOTE_TRIPLET(2/3),
    NINELET(2/9),
    QUARTER_NOTE(1),
    EIGHTH_NOTE(1/2),
    EIHTH_NOTE_TRIPLET(1/3),
    SIXTEENTH_NOTE(1/4),
    SIXTEENTH_NOTE_FIVELET(1/5),
    SIXLET(1/6),
    SEVENLET(1/7),
    THIRTYSECOND_NOTE(1/8),
    EIGHTH_NOTE_NINELET(1/9),
    TENLET(1/10);

    private final double duration;

    private RhythmType(double duration) {
        this.duration = duration;
    }

    public double getDuration(){
        return this.duration;
    }
}

File: flutter/hand_speed_calculator/lib/logic/rhythm_type.dart
enum RhythmType {
  DOT(1.5),
  WHOLE_NOTE(4.0),
  HALF_NOTE(2.0),
  EIGHTH_NOTE_FIVELET(2.0/5.0),
  QUARTER_NOTE_TRIPLET(2.0/3.0),
  NINELET(2.0/9.0),
  QUARTER_NOTE(1.0),
  EIGHTH_NOTE(1.0/2.0),
  EIHTH_NOTE_TRIPLET(1.0/3.0),
  SIXTEENTH_NOTE(1.0/4.0),
  SIXTEENTH_NOTE_FIVELET(1.0/5.0),
  SIXLET(1.0/6.0),
  SEVENLET(1.0/7.0),
  THIRTYSECOND_NOTE(1.0/8.0),
  EIGHTH_NOTE_NINELET(1.0/9.0),
  TENLET(1.0/10.0);

  final double _duration;

  const RhythmType(this._duration);

  double get getDuration => _duration;
}

